common/00_small_types_consolidated.cwt

```diff
@@ -184,6 +184,15 @@ faction = {
 	modifier = {
 		alias_name[modifier] = alias_match_left[modifier]
 	}
+
+	## cardinality = 0..1
+	triggered_faction_name = {
+		name = scalar
+
```

Comment:
Likewise, I think we should replace the scope here with the country scope:

`##replace_scope = { this = country root = country }`